README.md

```diff
@@ -22,8 +22,8 @@ This holds all of our best practices.
 ## Support
 
 1. [Trello](support/using_trello_for_support.md)
-1. [Professionalism](support/professionalism.md)
-1. [Slack Standup](support/slack_standup.md)
```

Comment:
NOTE: Markdown allows you to just put a number here to suggest using a numbered bullet.  Then you can move things around without having to edit numbers.  Please put it back the way it was.